lib/active_model/serializer/collection_serializer.rb

```diff
@@ -0,0 +1,44 @@
+module ActiveModel
+  class Serializer
+    class CollectionSerializer
+      NoSerializerError = Class.new(StandardError)
```

Comment:
I'm thinking maybe it should be a throw since it's essentially for flowcontrol

B mobile phone

> On Oct 7, 2015, at 5:11 AM, L. Preston Sego III <notifications@github.com> wrote:
> 
> In lib/active_model/serializer/collection_serializer.rb:
> 
> > @@ -0,0 +1,44 @@
> > +module ActiveModel
> > +  class Serializer
> > +    class CollectionSerializer
> > +      NoSerializerError = Class.new(StandardError)
> this seems like a weird place for this.
> Do we make new error types like this in other places?
> should there be a common place where the errors are all kept?
> 
> —
> Reply to this email directly or view it on GitHub.
>